client/gui/Fonts.cpp

```diff
@@ -352,27 +355,25 @@ void CBitmapHanFont::renderText(SDL_Surface * surface, const std::string & data,
 
 	SDL_LockSurface(surface);
 
-	for(size_t i=0; i<data.size(); i += Unicode::getCharacterSize(data[i]))
+	for(size_t i = 0; i < data.size(); i += Unicode::getCharacterSize(data[i]))
 	{
 		std::string localChar = Unicode::fromUnicode(data.substr(i, Unicode::getCharacterSize(data[i])));
 
-		if (localChar.size() == 1)
+		if(localChar.size() == 1)
 			fallback->renderCharacter(surface, fallback->chars[ui8(localChar[0])], color, posX, posY);
 
-		if (localChar.size() == 2)
+		if(localChar.size() == 2)
 			renderCharacter(surface, getCharacterIndex(localChar[0], localChar[1]), color, posX, posY);
 	}
 	SDL_UnlockSurface(surface);
 }
 
-CBitmapHanFont::CBitmapHanFont(const JsonNode &config):
-    fallback(new CBitmapFont(config["fallback"].String())),
-    data(CResourceHandler::get()->load(ResourceID("data/" + config["name"].String(), EResType::OTHER))->readAll()),
-    size(config["size"].Float())
+CBitmapHanFont::CBitmapHanFont(const JsonNode & config)
+	: fallback(new CBitmapFont(config["fallback"].String())), data(CResourceHandler::get()->load(ResourceID("data/" + config["name"].String(), EResType::OTHER))->readAll()), size(config["size"].Float())
```

Comment:
@DjWarmonger even if we don't want to enforce it we still need to decide on sane default for source we have now since most old code is badly formatted.